R/standalone-forcats.R

```diff
@@ -41,5 +41,20 @@ fct_rev <- function(f) {
   )
 }
 
+fct_expand <- function(factor, ...) {
```

Comment:
Can we rename the arguments to match `forcats::fct_expand(f, ..., after = Inf)`?

Reply:
I think we may be able to use a line like this: `levels(f) |> append(values = c(...), after = after)` to get the new levels.